NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/CountriesConsentViewController.cs

```diff
@@ -44,20 +44,20 @@ public override void ViewWillAppear(bool animated)
         public override void ViewWillDisappear(bool animated)
         {
             base.ViewWillDisappear(animated);
-            MessagingCenter.Unsubscribe<object>(this, MessagingCenterKeys.KEY_APP_RETURNS_FROM_BACKGROUND);
-            MessagingCenter.Unsubscribe<object>(this, MessagingCenterKeys.KEY_APP_WILL_ENTER_BACKGROUND);
+            MessagingCenter.Unsubscribe<object>(this, MessagingCenterKeys.KEY_APP_BECAME_ACTIVE);
+            MessagingCenter.Unsubscribe<object>(this, MessagingCenterKeys.KEY_APP_RESIGN_ACTIVE);
         }
 
         void AddObservers()
         {
-            MessagingCenter.Subscribe<object>(this, MessagingCenterKeys.KEY_APP_RETURNS_FROM_BACKGROUND, (object _) =>
+            MessagingCenter.Subscribe<object>(this, MessagingCenterKeys.KEY_APP_BECAME_ACTIVE, (object _) =>
             {
                 LogUtils.LogMessage(LogSeverity.INFO, "The user is seeing Countries Consent", null, GetCorrelationId());
             });
 
-            MessagingCenter.Subscribe<object>(this, MessagingCenterKeys.KEY_APP_WILL_ENTER_BACKGROUND, (object _) =>
+            MessagingCenter.Subscribe<object>(this, MessagingCenterKeys.KEY_APP_RESIGN_ACTIVE, (object _) =>
             {
-                LogUtils.LogMessage(LogSeverity.INFO, "The user left Countries Consent", null, GetCorrelationId());
+                LogUtils.LogMessage(LogSeverity.INFO, "The user is left Countries Consent", null, GetCorrelationId());
```

Comment:
Looks like your are bringing the typo back?

Reply:
I have corrected the error.